src/game/scenes/arena.c

```diff
@@ -1715,55 +1710,54 @@ int arena_create(scene *scene) {
             scene->gs->rec = omf_calloc(1, sizeof(sd_rec_file));
         } else {
             // release previous recording
-            sd_rec_free(local->rec);
+            sd_rec_free(scene->gs->rec);
         }
         sd_rec_create(scene->gs->rec);
-        local->rec = scene->gs->rec;
         for(int i = 0; i < 2; i++) {
             // Declare some vars
             game_player *player = game_state_get_player(scene->gs, i);
             log_debug("player %d using har %d", i, player->pilot->har_id);
-            local->rec->pilots[i].info.har_id = (unsigned char)player->pilot->har_id;
-            local->rec->pilots[i].info.pilot_id = player->pilot->pilot_id;
-            local->rec->pilots[i].info.color_3 = player->pilot->color_1;
-            local->rec->pilots[i].info.color_2 = player->pilot->color_2;
-            local->rec->pilots[i].info.color_1 = player->pilot->color_3;
-            local->rec->pilots[i].info.agility = player->pilot->agility;
-            local->rec->pilots[i].info.power = player->pilot->power;
-            local->rec->pilots[i].info.endurance = player->pilot->endurance;
-            local->rec->pilots[i].info.arm_speed = player->pilot->arm_speed;
-            local->rec->pilots[i].info.leg_speed = player->pilot->leg_speed;
-            local->rec->pilots[i].info.arm_power = player->pilot->arm_power;
-            local->rec->pilots[i].info.leg_power = player->pilot->leg_power;
-            local->rec->pilots[i].info.armor = player->pilot->armor;
-            local->rec->pilots[i].info.stun_resistance = player->pilot->stun_resistance;
-            memset(local->rec->pilots[i].info.name, 0, 18);
-            strncpy(local->rec->pilots[i].info.name, lang_get(player->pilot->pilot_id + 20), 18);
+            scene->gs->rec->pilots[i].info.har_id = (unsigned char)player->pilot->har_id;
+            scene->gs->rec->pilots[i].info.pilot_id = player->pilot->pilot_id;
+            scene->gs->rec->pilots[i].info.color_3 = player->pilot->color_1;
+            scene->gs->rec->pilots[i].info.color_2 = player->pilot->color_2;
+            scene->gs->rec->pilots[i].info.color_1 = player->pilot->color_3;
```

Comment:
3 & 1 wrong way around ?

Reply:
This is the same code you filed #935 against

Reply:
Aha. I can't keep these straight in my head.